contrib/builder/deb/arm/debian-jessie/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM armhf/debian:jessie
```

Comment:
We probably should move the `if` chain from the `Makefile` out so we have a
more standard "prefix" or "arch designation" (or both, really) and then it
could be used in more places and when we finally hear back from ARM about
what they'd officially like us to call each of these we can change it in
one main place.